build-packages.cmd

```diff
@@ -67,7 +49,7 @@ exit /b
 echo.
```

Comment:
For consistency we should decide on what should happen when the user types `-?`.
In PR dotnet/corefx#10231 I'm using the documentation that run.exe provides for the specific command which is one of the purposes of it, but sometimes I'm printing both, the one in the script and the one in run.exe.
The one in the script is to help developers understand what is happening and how the scripts are changing, like for example: [build.cmd](https://github.com/maririos/corefx/blob/56e4903a9c161e27e814366ff40ae3b1f93901d0/build.cmd#L12)